test_common/harness/testHarness.c

```diff
@@ -891,3 +900,29 @@ cl_device_id GetOpposingDevice( cl_device_id device )
     // Should never get here
     return NULL;
 }
+
+Version get_device_cl_version(cl_device_id device)
```

Comment:
There is function for checking CL version. Maybe we should not add this new one:
[get_device_version ](https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_common/harness/kernelHelpers.c#L734)

Reply:
There's also [this](https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_common/harness/errorHelpers.c#L765), [this](https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_conformance/api/test_api_min_max.c#L641), and [this](https://github.com/KhronosGroup/OpenCL-CTS/blob/master/test_conformance/computeinfo/main.c#L592). 

I'd prefer to add this simple one, and not over complicate the rest of the patch. Later we can cleanup the duplicates (I'm happy to do that once we merge the branches).

Reply:
Ok sounds good